packages/context/test/unit/context.ts

```diff
@@ -17,6 +17,13 @@ describe('Context', () => {
       expect(result).to.be.true();
     });
 
+    it('accepts a binding', () => {
+      const binding = new Binding('foo').to('bar');
```

Comment:
This code looks rather silly to me, as it reads  _new binding 'foo' to 'bar'_.

IMO, the API of `Binding` class was not designed to support usage outside of `Context` and we need to fix this problem before allowing components to contribute new bindings via `Binding` instance.